envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/ssl/EnvoySANValidationTest.kt

```diff
@@ -0,0 +1,74 @@
+package pl.allegro.tech.servicemesh.envoycontrol.ssl
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import pl.allegro.tech.servicemesh.envoycontrol.config.Ads
+import pl.allegro.tech.servicemesh.envoycontrol.config.EnvoyControlRunnerTestApp
+import pl.allegro.tech.servicemesh.envoycontrol.config.EnvoyControlTestConfiguration
+import pl.allegro.tech.servicemesh.envoycontrol.config.Xds
+
+class AdsEnvoySANValidationTest : EnvoySANValidationTest() {
```

Comment:
Do we need two tests for each mode? This makes our tests running longer. Maybe standard test with random mode (just like majority of our tests) will be enough?